src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs

```diff
@@ -336,29 +309,29 @@ private static bool ShouldSkipPath(string path)
             string name = Path.GetFileName(path).ToLowerInvariant();
             switch (name)
             {
-            case "cqo.dll":
-            case "fasttreenative.dll":
-            case "libiomp5md.dll":
-            case "libvw.dll":
-            case "matrixinterf.dll":
-            case "Microsoft.ML.neuralnetworks.gpucuda.dll":
-            case "MklImports.dll":
-            case "microsoft.research.controls.decisiontrees.dll":
-            case "Microsoft.ML.neuralnetworks.sse.dll":
-            case "neuraltreeevaluator.dll":
-            case "optimizationbuilderdotnet.dll":
-            case "parallelcommunicator.dll":
-            case "Microsoft.ML.Runtime.RunTests.dll":
-            case "scopecompiler.dll":
-            case "tbb.dll":
-            case "Internallearnscope.dll":
-            case "unmanagedlib.dll":
-            case "vcclient.dll":
-            case "libxgboost.dll":
-            case "zedgraph.dll":
-            case "__scopecodegen__.dll":
-            case "cosmosClientApi.dll":
-                return true;
+                case "cqo.dll":
+                case "fasttreenative.dll":
+                case "libiomp5md.dll":
+                case "libvw.dll":
+                case "matrixinterf.dll":
+                case "Microsoft.ML.neuralnetworks.gpucuda.dll":
```

Comment:
>Microsoft [](start = 22, length = 9)

considering what we push everything into lowercase, how it's even work....

Reply:
Lol. Good point. My hopes is this whole code just goes away very soon.